### PR TITLE
Fairmat 2023 pullback

### PR DESCRIPTION
**Pull request by sanbrock**

### CODE REVIEW

**Review by mkuehbach — CHANGES_REQUESTED** [review]
Hi @sanbrock and colleagues, content-wise this lgtm, however some changes I do not understand the rational behind but I would like to because when we pull this branch here back we should also update the nexus-fairmat-proposal. Maybe even the new NeXus release is out then soonish.
Ping me in scrum

File: base_classes/NXaperture.nxdl.xml
[the remark this answers is not in the document]
Comment (sanbrock) [reply base_classes/NXaperture.nxdl.xml:1]: All base classes modified by FAIRmat shall be kept here and **merge with  NIAC changes** rather then simply pulling over the new NIAC versions.

File: .github/workflows/ci.yaml
[the remark this answers is not in the document]
Comment (sanbrock) [reply .github/workflows/ci.yaml:1]: MK: fixit

File: contributed_definitions/NXelectronanalyser.nxdl.xml
[the remark this answers is not in the document]
Comment (sanbrock) [reply contributed_definitions/NXelectronanalyser.nxdl.xml:1]: fixit

**Comment by mkuehbach** [comment]
@domna and @mkuehbach idea exchange:
- Keep domain/technique-specific landing pages in fairmat-proposal
- But strip of repeated explanations and presentation of base classes, 
  changelog not so relevant, people should get to the appdef, slim
- in the long run rework the landing pages for tutorial-style (diataxis) view how these appdefs can be used

**Comment by mkuehbach** [comment]
Domain-specific landing pages, i.e. section number 3. - 10. from the side bar:
- [ ] Remove base section
- [ ] Add direct link to application definition (so that we do not need to navigate deeply with the side bar)

**Comment by domna** [comment]
@mkuehbach I checked the base classes and everything is fine (only minor docstring changes in NXbeam, which is fine).

File: contributed_definitions/NXelectronanalyser.nxdl.xml
[the remark this answers is not in the document]
Comment (mkuehbach) [reply contributed_definitions/NXelectronanalyser.nxdl.xml:1]: done

File: .github/workflows/ci.yaml
[the remark this answers is not in the document]
Comment (mkuehbach) [reply .github/workflows/ci.yaml:1]: done

**Comment by mkuehbach** [comment]
Thanks for checking and working on the proposal pages @domna,
the case with the dropped fields in NXaperture is suspicious though

**Comment by mkuehbach** [comment]
Refactoring of fairmat old proposal explanation pages completed, also fairmat logo updated

**Comment by domna** [comment]
> Thanks for checking and working on the proposal pages @domna,
> the case with the dropped fields in NXaperture is suspicious though

I don't see any dropped fields when I compare `fairmat_2023_pullback` to `fairmat_2023`. Or where there dropped w.r.t. to the main nexusformat repo?

File: base_classes/NXaperture.nxdl.xml
[the remark this answers is not in the document]
Comment (mkuehbach) [reply base_classes/NXaperture.nxdl.xml:1]: I checked binary differences between base_class dir contents of "nexusformat/definitions commitid 2bbcae4" (which is the newest state as of today (2023/07/24) which was not modified by use tagged add-missing s.th. branch) vs. "this PR  commitid 1d8af8f", there are no differences.

Also supportive evidence: as of today e.g. NXaperture with fairmat_2023_proposal vs fairmat_2023 the explicit enum for shape was already removed.

Based on all evidence above-mentioned and @domna analysis it seems that the indeed at some point added explicit enum shape and size are neither used by s.b. in fairmat currently nor are they included in the NIAC version.
For beam differences are white space locations.

I will herewith resolve the conversation but make one note:
As we have removed "NIAC" base classes from contributed
Changes on NIAC base classes i.e. changes should be done individually with individual short lived branches and fast PR merges. The only meeting point with the NIAC for this were the code camps.
If s.b. from fairmat experiences problems with the examples these should be articulated via scrum asap.

**Review by mkuehbach — APPROVED** [review]

**Pull request merged**